.github/actions/label-stale.yml

```diff
@@ -0,0 +1,20 @@
+name: "Label stale issues"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5
+        with:
+          days-before-issue-stale: 60
+          days-before-close: 65535 # We don't want to close issues in this action
+          stale-issue-label: stale
+          exempt-issue-labels: pinned,bug,enhancement
```

Comment:
Does this work with pinned? Do all pinned issues get a pinned label?

Reply:
Also, should we add another issue label such as "not stale" or whatever to ensure that we don't have to free important long term issues that are not bugs or enhancements from the bot every 2 months? Might never need it, but you never know.

Reply:
actually, don't think the `pinned` label is related to "pinning" an issue. I thought it was just a conventional name for "not stale". 
E.g. nextflow: https://github.com/nextflow-io/nextflow/issues/2702, but I think I also saw the "pinned" label in other places.

Reply:
I always interpreted it as “don’t mark as stale please”, not as having anything to do with pinned issues. But maybe that could be clearer?